commitizen/commands/check.py

```diff
@@ -126,14 +124,18 @@ def _filter_comments(msg: str) -> str:
         Returns:
             The filtered commit message without comments.
         """
-
-        lines = []
-        for line in msg.split("\n"):
-            if "# ------------------------ >8 ------------------------" in line:
-                break
-            if not line.startswith("#"):
-                lines.append(line)
-        return "\n".join(lines)
+        msg_lines = msg.split("\n")
```

Comment:
Not sure whether we want to do this. Look like we'll loop through lines twice and the logic looks more complicated